### PR TITLE
fix: let form mappings apply a behavior

### PR DESCRIPTION
**Pull request by lgray**

### CODE REVIEW

**Comment by lgray** [comment]
Ah - sorry this one requires things to get merged in dask-awkward, it is presently not possible in uproot to do it.

File: src/uproot/_dask.py [hunk 1]
```diff
@@ -772,7 +772,11 @@ def _get_meta_array(
     if form_mapping is not None:
         form = form_mapping(form)
     empty_arr = awkward.from_buffers(
-        form, 0, {"": b"\x00\x00\x00\x00\x00\x00\x00\x00"}, buffer_key=""
+        form,
+        0,
+        {"": b"\x00\x00\x00\x00\x00\x00\x00\x00"},
+        buffer_key="",
+        behavior=None if form_mapping is None else form_mapping.behavior,
     )
```
Comment (jpivarski) [inline src/uproot/_dask.py:780]: Nowadays, we can replace this whole thing with

```python
empty_arr = form.length_zero_array(behavior=None if form_mapping is None else form_mapping.behavior)
```

[defined here](https://github.com/scikit-hep/awkward/blob/4e01cc40e033cb9d06eee869cca73c4be3b0d53c/src/awkward/forms/form.py#L534-L547).

**Comment by lgray** [comment]
Looks like the new release of dask-awkward broke some existing tests.

**Comment by lgray** [comment]
@jpivarski could you re-run the tests on this branch, it should pass now?

**Review by jpivarski — APPROVED** [review]

**Pull request merged**